scripts/helper.js

```diff
@@ -145,17 +145,21 @@ const GetDepTree = ( name ) => {
 
 
 /**
- * Get all folders within a given path
+ * Get all modules within a given path.
+ * Module is folder with package.json
  *
  * @param  {string}  thisPath - The path that contains the desired folders
  * @param  {boolean} verbose  - Verbose flag either undefined or true
  *
  * @return {array}            - An array of names of each folder
  */
-const GetFolders = ( thisPath, verbose ) => {
+const GetModules = ( thisPath, verbose ) => {
 	try {
 		let folders = Fs.readdirSync( thisPath ).filter(
-				thisFile => Fs.statSync(`${ thisPath }/${ thisFile }`).isDirectory()
+				thisFile => {
+					let path = `${ thisPath }/${ thisFile }`;
+					return Fs.statSync(path).isDirectory() && Fs.existsSync(`${path}/package.json`)
```

Comment:
If we are going to refactor this function I think making it asynchronous would be really beneficial. Currently a lot of the code in `helper.js` is synchronous and as a good rule of thumb for any I/O this stuff should be async.

I like the readability changes but if we're updating this, we should make it _really_ nice

Reply:
Yes, there are a lot of  stuff which can be run asynchronous.

OK, i can "promisify" this function and change calling code.